pkg/traceql/test_examples.yaml

```diff
@@ -237,7 +237,11 @@ parse_fails:
   - '{} | bottomk(10) with(sample=0.1)'
   - '{} | rate() | topk(-1)'
   - '{} | rate() | bottomk(-1)'
-
+  - '{ parent = nil }'
```

Comment:
let's add valid nil queries under valid:?, we can add `{ nil = nil }, { nil != nil }, { span:name != nil }, and { nil != span:name }`.

Reply:
done. good call on this one. it caught a stringer error i didn't know existed b/c i hadn't handled the OpExists in UnaryOperation.String()